### ISSUE TITLE
Apologies - but where is the Recipe ? 

### ISSUE THREAD

**Issue by mromero-convoso** [sous-chefs/netplan#7]
Is this just a starter skeleton for a cookbook that was never finished? The Chef Supermarket has this cookbook passing 100% of its quality tests, which is confusing since I don't see a recipe.

**Comment by luishdez** [comment]
You're right. Actually I was reviewing this because there is a limitation with resolve.conf which only allow 3 dns providers I suffer this bug with k8s  (there is not notification for this)  and I wanted to inform about it but I didn't see any recipe as well.

**Comment by Xorima** [comment]
It's something that I planned on working on then never found time to do :( sorry, let me try to get back to this

**Comment by luishdez** [comment]
No problem :) we all know that sometimes it's really complicated to found time   Meanwhile could we clarify this cookbook is not developed yet in the readme?

**Comment by zedtux** [comment]
Would be cool @Xorima, my issue is open since then 😏 😛

**Comment by luishdez** [comment]
@Xorima do you have any blueprint or idea ? may be I can throw a rope. And start this recipe :)

**Comment by zedtux** [comment]
@luishdez I did one, have a look here: https://gitlab.com/pharmony/netplan.io-cookbook
